website/docs/components/stepper/indicator/partials/specifications/anatomy.md

```diff
@@ -2,7 +2,7 @@
 
 ### Step indicator
 
-![Step Indicator Anatomy](/assets/components/stepper/step-indicator-anatomy.png =212x*)
+![Step Indicator Anatomy](/assets/components/stepper/indicator/step-indicator-anatomy.png =212x*)
```

Comment:
The images should be appropriately sized in Figma before export that you shouldn't need "=212x*"